### PR TITLE
fix(chat-features): Add messageId and change id to participantId

### PR DESCRIPTION
**Pull request by he-patrick**
- messageId needs to be received so that reactions can be attached to specific messages
- participantId indicates that it is the participant's id rather than the message's id

Depends on: https://github.com/jitsi/lib-jitsi-meet/pull/2548

### CODE REVIEW

**Review by saghul — APPROVED** [review]
LGTM, we need to land the LJM change first though. @damencho can you PTAL too?

**Comment by saghul** [comment]
Code looks good, can you please fix the linting errors?

**Comment by he-patrick** [comment]
Fixed.

**Comment by he-patrick** [comment]
Let me know if that works 🙂

**Comment by saghul** [comment]
Unrelated test failure. Landing.

**Comment by damencho** [comment]
Nope, it is related

**Comment by damencho** [comment]
IFrameAPICommandsTest. testCommandSendChatMessage

**Comment by damencho** [comment]
We should figure it out what is the problem as it may break all tests, till now 2 out of 2 are failing.

File: react/features/chat/middleware.ts [hunk 1]
```diff
@@ -477,7 +487,7 @@ function _handleReceivedMessage({ dispatch, getState }: IStore,
 
         APP.API.notifyReceivedChatMessage({
             body: message,
-            id,
+            participantId,
```
Comment (saghul) [inline react/features/chat/middleware.ts:490]: @he-patrick This is what breaks the external API test. You need to adjust this: https://github.com/jitsi/jitsi-meet/blob/b2e06c1c94803144735c8b8f91e13bd01bc48503/modules/API/API.js#L1348

Please keep "from" there or applications will break.

**Review by saghul — CHANGES_REQUESTED** [review]

File: modules/API/API.js [hunk 2]
```diff
@@ -1814,9 +1814,9 @@ class API {
      * Notify external application of a participant, remote or local, being
      * removed from the conference by another participant.
      *
-     * @param {string} kicked - The ID of the participant removed from the
```
Comment (saghul) [inline modules/API/API.js:1817]: This looks like a merge conflict.

**Comment by saghul** [comment]
Jenkins please test this please.

**Review by saghul — APPROVED** [review]

**Comment by saghul** [comment]
🚀 🚀 🚀 🚀

**Pull request merged**